### PR TITLE
Add organ selection notebook

### PR DESCRIPTION
**Pull request by clarisse-lau**

### CODE REVIEW

**Comment by clarisse-lau** [comment]
Requesting final reviews, but please don't merge as I'd like to update the Colab link to the final production URL before merging :)

Link to rendered notebook: https://github.com/ncihtan/Community-Notebooks/blob/organ-type-notebook/HTAN/Python%20Notebooks/Identifying_HTAN_Data_Files_by_Organ_in_ISB_CGC.ipynb

**Comment by vthorsson** [comment]
Looks great @clarisse-lau !

**Review by vthorsson — APPROVED** [review]

**Comment by PozhidayevaDarya** [comment]
Looks great to me too :) Thanks @clarisse-lau!

**Review by PozhidayevaDarya — APPROVED** [review]

**Pull request merged**
